lib/origin_simulator/simulation.ex

```diff
@@ -17,6 +17,15 @@ defmodule OriginSimulator.Simulation do
     GenServer.call(server, :recipe)
   end
 
+  def route(server) do
+    GenServer.call(server, :route)
+  end
+
+  # for now, deal with minimum viable recipe: list containing a single recipe
+  def add_recipe(server, new_recipe) when is_list(new_recipe) and length(new_recipe) == 1 do
```

Comment:
Should we have a function that matches a recipe list with more than 1 and it returns a 406 (or appropriate status) with a message saying currently we have a limit of 1.

I know it's an edge case, but not sure how the app handles a situation with multiple recipes.

Reply:
Posting multiple recipes in a list, gets `406 Not Acceptable` for the time being.

Reply:
"The requested resource is only capable of generating content not acceptable according to the Accept headers sent in the request."

The 406 doesn't seem quite right based on the description for it. Maybe just a 400 bad request would be best

Reply:
I agree since there's no accept headers involved. The 406 has always been there as the standard response. Let's create a task to fix this later.